public/style/styles-compiler.scss

```diff
@@ -149,6 +149,7 @@ h1,h2,h3,h4,h5,h6, p {
 #change-location {
     @extend .large;
     @extend .m-b-xs;
+    height: 55px;
```

Comment:
Why not `em`? pixel won't adjust depending on the font size that changes depending on screen width.